lib/stack_master/commands/apply.rb

```diff
@@ -121,7 +121,7 @@ def stack_options
         {
           stack_name: stack_name,
           parameters: proposed_stack.aws_parameters,
-          capabilities: ['CAPABILITY_IAM'],
+          capabilities: ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],
```

Comment:
As does setting `CAPABILITY_IAM`

Alternatively these could be enabled with command line flags.